metrics/src/main/java/tech/pegasys/pantheon/metrics/MetricCategory.java

```diff
@@ -18,7 +18,8 @@
   JVM("jvm", false),
   PROCESS("process", false),
   BLOCKCHAIN("blockchain"),
-  SYNCHRONIZER("synchronizer");
+  SYNCHRONIZER("synchronizer"),
+  NETWORK("network");
```

Comment:
Not for this PR, but I'm really feeling like we need hierarchical categories.  The `PEERS` category is both worthy of it's own category but also very highly network related.  Having `NETWORK->PEERS` would make a lot of sense.

Reply:
I'll keep it in mind if I ever mess with categories.  May be deep.